server/test/chokidar-test.ts

```diff
@@ -111,7 +111,7 @@ describe('chokidar', function() {
     });
   });
 
-  it('notifies about file changes', function() {
+  it('notifies about file changes', async function() {
```

Comment:
that shouldn't matter as the `async` is on the lambda function below, and `withWatcher` returns a Promise